packages/bpk-component-banner-alert/src/BpkBannerAlert.module.scss

```diff
@@ -62,8 +64,7 @@
 
   &__header {
     display: flex;
-    padding: $bpk-banner-alert-header-padding-y
```

Comment:
Guessing there will be a corresponding PR open to deprecate this token in Foundations?

Reply:
Yes!